### ISSUE TITLE
http_get should support headers and params too.

### ISSUE THREAD

**Issue by alitrack** [quackscience/duckdb-extension-httpclient#7]
here is an example, how to translate to http_get?

```bash
curl https://api.openai.com/v1/models \
-H "Content-Type: application/json" \
-H "Authorization: Bearer sk-xxx"
```

**Comment by lmangani** [comment]
Hello @alitrack 👋  the same syntax as the POST request should work.:

````sql 
     SELECT http_get(
          'https://httpbin.org/delay/0',
          headers => MAP {
            'accept': 'application/json',
          },
          params => MAP {
          }
    ) AS res
```

**Comment by ahuarte47** [comment]
I think it is not supported:
![image](https://github.com/user-attachments/assets/6276c684-095f-46b5-ba52-66058c1762ff)

Maybe we should create other function `http_get` function with those extra parameters. Just to do not change current function. What do you think?

**Comment by lmangani** [comment]
My bad! We could declare multiple versions of the function with the extra parameters or consider named parameters to make them optional.

**Comment by lmangani** [comment]
@alitrack the feature has been implemented by @ahuarte47 (gracias!) and will be available for DuckDB v1.1.3 tomorrow. Let us know if you get around testing it and/or have other feedback.

Closing as implemented/resolved.
